applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -386,6 +407,13 @@ int bt_mgmt_init(void)
 		return ret;
 	}
 
+	if (IS_ENABLED(CONFIG_TESTING_BLE_ADDRESS_RANDOM)) {
+		ret = static_addr_random();
+		if (ret) {
+			return ret;
+		}
+	}
+
 	if (IS_ENABLED(CONFIG_SETTINGS)) {
 		ret = settings_load();
```

Comment:
```suggestion
		if (!IS_ENABLED(CONFIG_TESTING_BLE_ADDRESS_RANDOM)) {
		    ret = settings_load();
```
and then remove the code you added below. If we never load in the bonding information we don't need to clear it